mysql/datadog_checks/mysql/const.py

```diff
@@ -235,6 +235,9 @@
     'wsrep_flow_control_sent': ('mysql.galera.wsrep_flow_control_sent', MONOTONIC),
     'wsrep_cert_deps_distance': ('mysql.galera.wsrep_cert_deps_distance', GAUGE),
     'wsrep_local_send_queue_avg': ('mysql.galera.wsrep_local_send_queue_avg', GAUGE),
```

Comment:
Another important metric is missing here: `wsrep_local_cert_failures`: 
`Total number of local transactions that failed certification test` (cf: https://galeracluster.com/library/documentation/galera-status-variables.html#wsrep-local-cert-failures).
It's about the "conflicts" that we never want to have (because it refuses commits) on production cluster.
So, we should add it too.